babel/src/main/codegen/config.fmpp

```diff
@@ -578,9 +578,14 @@ data: {
     ]
 
     # Custom identifier token.
-    # Example: "< IDENTIFIER: (<LETTER>|<DIGIT>)+ >".
-    customIdentifierToken: "< IDENTIFIER: (<LETTER>|<DIGIT>)+ >"
-
+    # 
+    # PostgreSQL allows letters with diacritical marks and non-Latin letters 
+    # in the beginning of identifier and additionally dollar sign in the rest of identifier. 
+    # See https://github.com/postgres/postgres/blob/master/src/backend/parser/scan.l
+    #
+    # MySQL allows digit in the beginning of identifier
+    customIdentifierToken: "< IDENTIFIER: (<LETTER>|<DIGIT>|[\"\\200\"-\"\\377\"]) (<LETTER>|<DIGIT>|<DOLLAR>|[\"\\200\"-\"\\377\"])* >"
```

Comment:
What do they mean?`\200-\377`?
Could you please give them mapping table link in JIRA?

Because only the dollar symbol is described in the title .

Reply:
@JiajunBernoulli 
It is letters with diacritical marks and non-Latin letters.

From https://github.com/postgres/postgres/blob/master/src/backend/parser/scan.l
```
ident_start		[A-Za-z\200-\377_]
ident_cont		[A-Za-z\200-\377_0-9\$]

identifier		{ident_start}{ident_cont}*
```
I will rename jira ticket and add `diacritical marks and non-Latin letters` to title. description has it already

Reply:
Is it octal?

Here is char mapping link: https://learn.microsoft.com/zh-cn/office/vba/language/reference/user-interface-help/character-set-128255
Octal  |  Decimal  |  
377     |  255         |   ÿ

Reply:
Yes, it is octal.

Reply:
I hope `octal` is emphasized in the comments of the code.

And no other suggestions.

Reply:
@JiajunBernoulli I added comment about octal.

Reply:
@JiajunBernoulli, can I merge it, or do you still have some concerns?

Reply:
Sorry, I missed your message two days ago.

Now I approved it.